reports/filesDataExporter/handlers.go

```diff
@@ -0,0 +1,176 @@
+package filesDataExporter
+
+import (
+	"bytes"
+	"context"
+	"sort"
+
+	"github.com/go-openapi/runtime"
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+	"github.com/rs/zerolog"
+
+	"github.com/iryonetwork/wwm/gen/storage/client/operations"
+	"github.com/iryonetwork/wwm/gen/storage/models"
```

Comment:
could not import github.com/iryonetwork/wwm/gen/storage/models (cannot find package "github.com/iryonetwork/wwm/gen/storage/models" in any of:
	/tmp/tmp.ZnHXsVY6xN/src/github.com/iryonetwork/wwm/vendor/github.com/iryonetwork/wwm/gen/storage/models (vendor tree)
	/usr/local/go/src/github.com/iryonetwork/wwm/gen/storage/models (from $GOROOT)
	/tmp/tmp.ZnHXsVY6xN/src/github.com/iryonetwork/wwm/gen/storage/models (from $GOPATH))